src/Sylius/Component/Core/Model/OrderItem.php

```diff
@@ -57,4 +57,24 @@ public function equals(BaseOrderItemInterface $item)
             && $item->getVariant() === $this->variant
         );
     }
+
+    /**
+     * Returns sum of neutral and non neutral tax adjustments on order item and total tax of units.
+     *
+     * {@inheritdoc}
+     */
+    public function getTaxTotal()
```

Comment:
@mmckelviereiss we're thinking about extracting this to a separate service, because you can have different strategies, but we decided to start with this simple implementation and think about improvements later on :) You can easily override this method in your model and implement the logic you need, so I believe it can stay on a model level for now 😉